### DISCUSSION TITLE
Documentation to setup butler for failure email alerts

### DISCUSSION THREAD

**Discussion by DZB3M0** [ptarmiganlabs/butler#1130] [General]
Documentation to setup butler for failure email alerts

**Comment by mountaindude** [comment]
All info is available at [butler.ptarmiganlabs.com](https://butler.ptarmiganlabs.com).

There you find links to both this GitHub repo (which is where pre-compiled binaries for Windows, macOS, Linux) are found, as well as to the more detailed doc pages at butler.ptarmiganlabs.com.

For example, click Documentation > Getting started and you get several pages giving you an overview of how Butler works, what's needed to set it up etc. 

Butler has a lot of different features that can be individually turned on/off, the reload failed alerting feature is described [here](https://butler.ptarmiganlabs.com/docs/getting-started/setup/reload-alerts/).

[This blog post](https://ptarmiganlabs.com/the-lazy-qlik-developer-2-setting-up-butler-sos/) at ptarmiganlabs.com may also be useful. It describes how to set up [Butler SOS](https://butler-sos.ptarmiganlabs.com), which is a sibling tool to Butler.  
Setting up Butler is pretty similar to (but not identical with) setting up Butler SOS, so that blog post may be useful too.
